pkg/scheduler/ugm/manager.go

```diff
@@ -414,7 +416,7 @@ func (m *Manager) clearEarlierSetUserWildCardLimits(newUserWildCardLimits map[st
 					log.Log(log.SchedUGM).Debug("Need to clear earlier set configs for user because wild card limit has been applied earlier",
 						zap.String("user", ut.userName),
 						zap.String("queue path", queuePath))
-					ut.setLimits(hierarchy, nil, 0, false, true)
+					ut.clearLimits(queuePath)
```

Comment:
4th & 5th arguments in setLimits() has significance. It can't be ignored.

Reply:
I believe this should have been caught by unit tests. I'll file a JIRA later to properly cover setLimits/clearLimits.